src/snowflake/cli/plugins/snowpark/package/commands.py

```diff
@@ -105,7 +113,13 @@ def package_create(
         install_packages = deprecated_install_option
 
     if (
-        type(lookup_result := lookup(name=name, install_packages=install_packages))
+        type(
+            lookup_result := lookup(
+                name=name,
+                install_packages=install_packages,
+                allow_native_libraries=allow_native_libraries,
+            )
+        )
         in [
             NotInAnaconda,
             RequiresPackages,
```

Comment:
I believe that `isinstance(look_up, (NotInAnaconda, RequiredPackages))` suits better

Reply:
also this lookup_result could be assigned using regular = outside if.
it would be easier to read

Reply:
Changed